### PR TITLE
Move SQM to AI for telemetry in AppDesigner and Editors projects

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

**Comment by mavasani** [comment]
Tagging @dotnet/roslyn-analysis for review

File: src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj [hunk 1]
```diff
@@ -258,9 +258,6 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\Common\IVsSqm.vb">
```
Comment (davkean) [inline src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj]: Can we delete this now?

**Comment by mavasani** [comment]
I verified that I get telemetry events on opening the VB property page. However, I also get the below NRE (even before my changes). Is this known issue?

nre_property_pages

File: src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj (hunk 1 above)
Comment (davkean) [inline src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj]: Oh you did, ignore me.

Comment (mavasani) [inline src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj]: Yes, I thought I deleted this file..

**Comment by davkean** [comment]
Yep that's expected at the moment - that's because the new project system is handling csproj/vbpro and the AppDesigner pages don't yet work.

:+1:

**Comment by srivatsn** [comment]
👍

**Pull request merged**